code/modules/cm_aliens/structures/egg.dm

```diff
@@ -208,7 +208,7 @@
 		return
 
 	user.animation_attack_on(src)
-	if(W.attack_verb.len)
+	if(W.attack_verb)
```

Comment:
Do length(W.attack_verb) instead. Will stop the runtime and insure we have at least one attack_verb to choose from in the to_chat()